mappings/com/mojang/blaze3d/platform/GLX.mapping

```diff
@@ -0,0 +1,218 @@
+	FIELD field_30532 source1Rgb I
+	FIELD field_30533 source2Rgb I
+	FIELD field_30534 operand0Rgb I
+	FIELD field_30535 operand1Rgb I
+	FIELD field_30536 operand2Rgb I
+	FIELD field_30537 combineAlpha I
+	FIELD field_30538 source0Alpha I
+	FIELD field_30539 source1Alpha I
+	FIELD field_30540 source2Alpha I
+	FIELD field_30541 operand0Alpha I
+	FIELD field_30542 operand1Alpha I
+	FIELD field_30543 operand2Alpha I
+	FIELD field_30544 useSeparateBlendExt Z
+	FIELD field_30545 isOpenGl21 Z
+	FIELD field_30546 usePostProcess Z
+	FIELD field_30547 useVbo Z
+	FIELD field_30548 isAmdVboSupported Z
+	FIELD field_30549 arrayBuffer I
+	FIELD field_30551 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_30552 fboMode Lnet/minecraft/class_6125$class_6126;
+	FIELD field_30553 hasShaders Z
+	FIELD field_30554 useShaderArb Z
+	FIELD field_30555 useMultitextureArb Z
+	FIELD field_30556 useTexEnvCombineArb Z
+	FIELD field_30557 separateBlend Z
+		COMMENT States whether OpenGL 1.4 features are supported.
+		COMMENT
+		COMMENT <p>
+		COMMENT \tThis includes
+		COMMENT \t<ul>
+		COMMENT \t\t<li>Separate Blend Functions</li>
+		COMMENT \t\t<li>Automatic mipmap generation</li>
+		COMMENT \t\t<li>Stencil wrapping</li>
+		COMMENT \t</ul>
+		COMMENT \tand many more.
+		COMMENT </p>
+	FIELD field_30558 isNvidia Z
+	FIELD field_30559 capsString Ljava/lang/String;
+	FIELD field_30560 cpuInfo Ljava/lang/String;
+	FIELD field_30561 useVboArb Z
+	FIELD field_30562 isAmd Z
+	FIELD field_30563 framebuffer I
+	FIELD field_30564 renderbuffer I
+	FIELD field_30565 colorAttachment I
+	FIELD field_30566 depthAttachment I
+	FIELD field_30567 completeFramebuffer I
+	FIELD field_30568 incompleteFramebufferAttachment I
+	FIELD field_30569 incompleteFramebufferAttachmentMiss I
+	FIELD field_30570 incompleteFramebufferAttachmentDraw I
+	FIELD field_30571 incompleteFramebufferAttachmentRead I
+	FIELD field_30572 advancedOpenGlEnabled Z
+	FIELD field_30573 linkStatus I
+	FIELD field_30574 compileStatus I
+	FIELD field_30575 vertexShader I
+	FIELD field_30576 fragmentShader I
+	FIELD field_30577 textureUnit I
+	FIELD field_30578 lightmapTextureUnit I
+	FIELD field_30579 texture I
+	FIELD field_30580 combine I
+	FIELD field_30581 interpolate I
+	FIELD field_30582 primaryColor I
+	FIELD field_30583 constant I
+	FIELD field_30584 previous I
+	FIELD field_30585 combineRgb I
+	FIELD field_30586 source0Rgb I
+	METHOD method_31782 setupContext ()V
+	METHOD method_31783 deleteShader (I)V
+		COMMENT Deletes a shader object
+		ARG 0 shader
+	METHOD method_31784 multiTexCoord2f (IFF)V
+		COMMENT Sets the current texture coordinates
+	METHOD method_31785 getProgram (II)I
+		COMMENT Returns a parameter from a program object
+		ARG 0 program
+		ARG 1 param
+	METHOD method_31786 renderbufferStorage (IIII)V
+		COMMENT Establishes data storage, format and dimensions of a renderbuffer object's image
+	METHOD method_31787 framebufferTexture2DEXT (IIIII)V
```

Comment:
DEXT? Wouldnt it be just `framebufferTexture2D`

Reply:
because it’s unobf 
right?

Reply:
Cause upstream calls it that in GLStateManager